README.md

```diff
@@ -580,6 +701,18 @@ cancellation of all outstanding activities.
 The `shutdown()` invocation will wait on all activities to complete, so if a long-running activity does not at least
 respect cancellation, the shutdown may never complete.
 
+#### Testing
+
+Unit testing an activity or any code that could run in an activity is done via the
+`temporalio.testing.ActivityEnvironment` class. Simply instantiate this and any callable + params passed to `run` will
+be invoked inside the activity context. The following are attributes/methods on the environment that can be used to
+affect calls activity code might make to functions on the `temporalio.activity` package.
+
+* `info` property can be set to customize what is returned from `activity.info()`
+* `on_heartbeat` property can be set to handle `activity.heartbeat()` calls
```

Comment:
You might want to mention that when a worker runs an activity it throttles sending out heartbeats to the server so that users don't rely on this function in their tests.

Reply:
Can document, but can you clarify what you mean by:

> users don't rely on this function in their tests

This is the one literally invoked for each heartbeat call unthrottled and I'd expect they can rely on that. Or are you saying so they don't improperly assume every heartbeat is heavy and that people call it quickly a lot so `on_heartbeat` should not be assumed to be 1:1 with server heartbeat?

Reply:
the latter, thanks for clarifying.

Reply:
Will document when I send in PR for #120